app/Locale/Slovak.php

```diff
@@ -308,11 +302,11 @@ public function get_locale(): array
 			'SETTINGS_SUCCESS_IMAGE_OVERLAY' => 'EXIF-Overlay nastavenia aktualizované',
 			'SETTINGS_SUCCESS_PUBLIC_SEARCH' => 'Verejné vyhľadávanie bolo aktualizované',
 			'SETTINGS_SUCCESS_LICENSE' => 'Prednastavená licencia aktualizovaná',
-			'SETTINGS_SUCCESS_CSS' => 'CSS aktualizované',
-			'SETTINGS_SUCCESS_UPDATE' => 'Nastavenia úspešne aktualizované',
```

Comment:
Why this deletion?

Reply:
Moved to be aligned with the other languages.